codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -40,6 +54,7 @@
      * Creates a new handler with the default compression level (<tt>6</tt>),
      * default window size (<tt>15</tt>) and default memory level (<tt>8</tt>).
      */
+    @Deprecated
     public HttpContentCompressor() {
```

Comment:
I did revert the changes to this constructor as sub-classes of this class might depend on the `determineWrapper` method to be called and delegate to this constructor